packages/server/src/tests/utilities/TestConfiguration.ts

```diff
@@ -372,21 +373,20 @@ export default class TestConfiguration {
       admin,
     }
     await sessions.createASession(_id, {
-      sessionId: "sessionid",
+      sessionId: this.sessionIdForUser(_id),
       tenantId: this.getTenantId(),
       csrfToken: this.csrfToken,
     })
     const resp = await db.put(user)
+    await cache.user.invalidateUser(_id)
     return {
       _rev: resp.rev,
       ...user,
     }
   }
 
   async createUser(user: Partial<User> = {}): Promise<User> {
-    const resp = await this.globalUser(user)
-    await cache.user.invalidateUser(resp._id!)
-    return resp
+    return await this.globalUser(user)
```

Comment:
Given that this is now down to just a call to `globalUser`, I think I will do a follow up PR that unifies the uses of this function and gives it a better name. The cache invalidation was the last holdout, and that's now moved into `globalUser` because I feel like it's more correct to have it there.

Reply:
Yeap, this definitely could be updated, right now its a little confusing.